### ISSUE TITLE
Results reproduction

### ISSUE THREAD

**Issue by Pavelrst** [princeton-vl/RAFT#5]
Hi, thanks for sharing the code! I would like to reproduce the results of the paper and I have three questions. Let's say I want to fine-tune the chairs+things checkpoint on Sintel dataset and compare my results to the results in the paper.

1) What are the relevant metrics? In the paper I see two different tabels: Table 4 mentions 'accuracy'. Table 1 does not mention any metrics, just numbers. Looking at Table 2 (Kitti dataset) I can see f1-epe and f1-all metrics. Fig.6 says "Accuracy is measured by the EPE on the Sintel(train) final pass
after training on C+T". So what are the metrics to measure the performance of the fine-tuned model? Looking at the evaluation function, looks like the relevant metric is EPE. AM I right?

2) Looking at the Sintel dataset, I see two sub sets: Clean and Final. The fine-tuning of the model on the Sintel dataset is done by training on both Clean and Final for 60000 steps. According to Table 1, a model which trained on Chairs+Things is evaluated on Sintel (separately on Clean and Final) the numbers (EPE I guess?) are - Clean: 1.63, Final: 2.83. When a Sintel-fine-tuned model evaluated, the numbers are: 2.42, 3.39. My question is the following: On which dataset a Sintel-fine-tuned model was evaluated?

3) The number of fine-tuning steps are basically the number of batches which were fed to the model during the ft process, am I right? If I reduce the size of the batch, should I scale the number of steps as well? i.e. Sintel ft is 60,000 with batch size 4. If want to use batch size 2, the number of steps should be 120,000?

Thanks a lot!

**Comment by zachteed** [comment]
Hi Pavel, 

1. Most results are evaluated using end-point-error (EPE), which is the average euclidean distance between the predicted and ground truth flow vectors. The F1 score is unique to the KITTI dataset, and is defined as the percentage of the flow vectors which are either within 3px or within 5% of the ground truth flow field.

2. The Sintel test results are the numbers on the official test test. The ground truth flow is not publicially available, and these results are from our submission to the Sintel leaderboard evaluation [http://sintel.is.tue.mpg.de/results](Sintel).

3. Yes that's correct. If you reduce the batch size you will need to increase the number of training steps. You will probably also need to reduce the learning rate.
